### DISCUSSION TITLE
Streaming Issues

### DISCUSSION THREAD

**Discussion by ooom416354** [GerryDazoo/Slinger#453] [Q&A]
Hi All, 

Post N9 everything's worked well, just the past few days trying to stream via VLC hasn't been working.  It spins but I'm not getting anything back.  Everything is on the same network, no concerns there, I'm using a 500 with all traffic blocked.  I've tried using the IP address of the Pi running Slinger as well and to no avail.  Here's my logs, anything in particular looking off? (192.168.86.94 is the IP address of the PC I'm running VLC on).

Starting remote control stream handler for  ('192.168.86.1', 54978) to port 8029
Remote Control Connected
127.0.0.1 - - [22/Dec/2022 08:18:10] "^[[37mGET /Remote HTTP/1.1^[[0m" 200 -
Exiting Remote Control Stream Handler for ('192.168.86.1', 54978)
12/22/2022, 08:18:11   RemoteControl connection from ('192.168.86.1', 54984)

Starting remote control stream handler for  ('192.168.86.1', 54984) to port 8029
Remote Control Connected
127.0.0.1 - - [22/Dec/2022 08:18:11] "^[[33mGET /favicon.ico HTTP/1.1^[[0m" 404 -
Exiting Remote Control Stream Handler for ('192.168.86.1', 54984)
Hacker Alert. Invalid Request from  ('192.168.86.94', 51946)
Hacker Alert. Invalid Request from  ('192.168.86.94', 51947)
Hacker Alert. Invalid Request from  ('192.168.86.94', 51948)
Hacker Alert. Invalid Request from  ('192.168.86.94', 51949)
12/22/2022, 08:26:45   Streaming connection from ('192.168.86.94', 51950)

No changes to the config file either and the remote control works when connecting to that link.

Thanks.

**Comment by ooom416354** [comment]
FYI URL to connect via VLC is as follows (IP of the Pi running Slinger is 192.168.86.95 and I changed the port to 8028, it's open on my router but I'm connecting internally to test on the same network so that shouldn't matter):

http://192.168.86.95:8028/slingbox

Config is as follows:

[SLINGBOX]
sbtype="350/500/M1"
password=c0****

[SERVER]
port=8028
maxstreams=10
enableremote=yes

[REMOTE]
include=tivo_remote.txt

**Comment by GerryDazoo** [comment]
Your using the wrong URL in VLC "GET /Remote "
Remote isn't for remote access it's to bring up the Remote Control in a browser.
ooman's answer is correct.

**Reply by ooom416354** [reply]
Hi Gerry, I am Ooom, I just replied to my original message with the config.  I'm using the following to connect:

http://192.168.86.95:8028/slingbox

Just nothing is happening in VLC, nothing changed since it worked a few days ago, and I'm not receiving any errors in n VLC (I know if I use the wrong connection string it'll error).  Just not sure what could be going on here, I'd expect something to start to stream you know?

**Reply by GerryDazoo** [reply]
we need to see the server output now. please post it all from server startup.

**Reply by ooom416354** [reply]
So I redid the install on the Pi with the latest build and it didn't work at first. So I stopped the service and started it up again and it worked.

I had always has it running automatically on a restart of the Pi but might have missed a step somewhere.  Is there a way to start it up automatically or check why it requires a service restart after a reboot of the Pi?
